apps/admin-dashboard/app/routes/_dashboard.onboarding-sessions.tsx

```diff
@@ -100,6 +106,7 @@ async function listOnboardingSessions({
     return {
       ...row,
       date: dayjs(row.date).format('MM/DD/YY'),
+      ambassadorName: row.ambassadorName || '',
```

Comment:
Considers case when there is no ambassadorName stored (for old onboarding sessions)